ionoscloud/resource_server_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build compute || all || server || enterprise
-// +build compute all server enterprise
+//go:build compute || all || server || enterprise || cube
```

Comment:
This tag was added because there is a `TestAccServerCubeServer` test case defined in this file, since a `cube` server can also be created through the `ionoscloud_server` resource by setting the type attribute.